androgee/cogs/spam_badwords.py

```diff
@@ -0,0 +1,83 @@
+from discord.ext import commands
+from androgee.init import MOD_ROLE_ID, MOD_ROLE_NAME, swear_list, COMMAND_PREFIX
+
+
+def isbad(word: str) -> bool:
+    if word in swear_list:
+        return True
+    return False
+
+
+class BadWords(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.Cog.listener()
+    async def on_ready(self) -> None:
+        print(f"We is logged in as {self.bot.user}")
+
+    @commands.has_any_role(MOD_ROLE_NAME, int(MOD_ROLE_ID))
+    @commands.command(name="unlock")
+    async def unlock(self, ctx) -> None:
```

Comment:
What does the `-unlock` command have to do with `spam_badwords`? That feels like just a general admin command to me

Reply:
when the bot detects spam it locks the channel meaning only the admins can send messages in this channel